app/domain/search/audit_filter.rb

```diff
@@ -0,0 +1,42 @@
+class Search
+  class AuditFilter
+    AUDIT_STATUSES_IDENTIFIERS = [
+      :audited,
+      :non_audited,
+    ].freeze
+
+    attr_accessor :status
+
+
+    def initialize(status)
+      self.status = status ? status.to_sym : nil
```

Comment:
Why do we need to handle `nil`? I was thinking the `nil` case would be handled by not calling the `audit_status=` method so that `filters` doesn't include any instance of `AuditFilter`. What do you think?